src/pages/Charcuterie.js

```diff
@@ -53,244 +65,248 @@ export default () => {
     multiselect: { option1: false, option2: false, selected: false, disabled: false },
   })
   const [textAreaValue, setTextAreaValue] = useState('')
+  const [progress, setProgress] = useState(0)
 
   return (
     <>
-      <Button color="secondary" width="flex" href={`javascript:(${toggleTheme})()`}>
-        Toggle Theme
-      </Button>
-      <P>
-        Theme switcher. Drag the bookmarklet button from the page to your Bookmarks Toolbar. It
-        should appear on the toolbar
-      </P>
-
-      <H1>Charcuturie</H1>
-      <>
-        <H1>This is an h1.</H1>
-        <H2>This is an h2.</H2>
-        <H3>This is an h3.</H3>
-        <P>
-          <A href="https://en.wikipedia.org/wiki/Charcuterie">Charcuterie</A> most often consists of
-          a variety of meats and cheeses, often paired with crackers, fruit, nuts, and spreads. An
-          ideal charcuterie board has a good balance of flavors and textures and has foods that
-          contrast and complement each other's taste. It's some really long text. I'm really writing
-          this way later than I should be. Is this what it's like to sell your soul to nwPlus?{' '}
-        </P>
-        <br></br>
-        <A href="https://nwplus.io">This is a link.</A>
-        <br></br>
-        <br></br>
-        <A color="primary" bolded href="https://nwplus.io">
-          This is a hacker app link.
-        </A>
-        <br></br>
-        <br></br>
-        <A color="primary" href="https://nwplus.io">
-          This is an unbolded primary color link.
-        </A>
+      <Container>
```

Comment:
I don't think we need to create a container here. I'm happy for the progress bar to be inline at the bottom of the charcuterie page

Reply:
![image](https://user-images.githubusercontent.com/56183965/100530399-73ad4700-31a6-11eb-85cd-e49cd7beb8c7.png)

Do you mean something like this?

Reply:
Oh i didn't realize it has absolute positioning. Maybe the component doesnt need positioning? And the consumer can choose how to position it?

Reply:
Yea I think it's fine if we stick it outside the container for now. After you add the form wrapper in your other PR #138, can you position the sidebar like how it is in the designs and update the usage in this file? or can you somehow make them into one component so it can be easily used?
<img width="226" alt="Screen Shot 2020-11-28 at 6 40 24 PM" src="https://user-images.githubusercontent.com/38872354/100530631-30080c80-31a9-11eb-974d-305249dc9645.png">

Reply:
Oh yeah good point. You mean I should remove the positioning in the progress bar component and the consumer would make a wrapping component around it that has the positioning styling right?

Reply:
wait actually I think we should still have positioning, just because in all of its usecases, it needs to have position set as absolute

Reply:
scratch what i said about integrating the wrapper and the progress bar - we'll style it later
let's just leave position set on the progress bar